packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -506,7 +509,9 @@ export const CheckoutStepAreaWrapper = styled.div`
 	}
 
 	@media ( ${ ( props ) => props.theme.breakpoints.tabletUp } ) {
-		margin-bottom: 0;
+		&.checkout__step-wrapper--last-step {
```

Comment:
This was actually a bug in the implementation. The last step has a `margin-bottom` when at mobile width so it leaves space for the fixed position submit button, and that margin should be removed at wider widths when the submit button is not fixed. However, the removal was putting the `margin-bottom` on the wrong element, meaning that the last step always had the tall bottom margin.